maincontroller.cpp

```diff
@@ -238,10 +238,15 @@ void MainController::processJudgingData(QJsonDocument judgingData) {
         for (int j = 0; j < rankedTeam->getNumProblems(); j++) {
             Model::RankedProblem *rankedProblem = rankedTeam->getProblem(j);
             ResultProblem problem;
-            problem.numTries = rankedProblem->tries;
             problem.state = rankedProblem->problemState;
             problem.problemId = rankedProblem->shortname;
-            problem.time = rankedProblem->timeLastTry;
+            if (rankedProblem->problemState == SOLVED){
```

Comment:
Not for now but for later: this whole code is duplicated three times (by me, I know...). Maybe we can make a function from it.

Reply:
Yeah sure I notice that, but I was focusing resolving the bug.

Reply:
Probably if we are going to use the event feed this might change anyway, so maybe do it after that.

Reply:
Yeah for sure, I will try to remove all the duplicated code and generify them.